docs/faq/general.md

```diff
@@ -23,7 +23,32 @@ In other words, it is a long e as in the word decentralized.
 
 ---
 
+#### **3. How was the Decred premine distributed?**
+
+The premine consisted of 8% of the total supply of 21 million coins (1.68 million coins).[^1] This was split equally between compensation for bring-up costs and an “airdrop”.
```

Comment:
I think the first sentence should be in the present tense (The premine consistS of...)